real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/nodes/sensors/RBSMNode.java

```diff
@@ -232,7 +232,13 @@ else if (headerNumber == RBSerialMessage.getHeaderByte("DEVICE_ID")){
 			messagePubDeviceID.publish(new DeviceIDMessage(message.getDataWord()));
 		}
 		else if (headerNumber == RBSerialMessage.getHeaderByte("RBSM_MID_MEGA_BRAKE_STATE")){
-			messagePubBrakeState.publish(new BrakeStateMessage(message.getDataWord()));
+			boolean brakesDown = false;
+
+			if (message.getDataWord() == 1) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Avoid using Literals in Conditional Statements](https://www.codacy.com/app/Mr-Trevor-Decker/RoboBuggy/file/2343306767/issues/source?bid=2938665&fileBranchId=3212596#l237)